tensorvis/tensorvisio/src/processors/tensorfield3dimport.cpp

```diff
@@ -57,33 +60,30 @@ TensorField3DImport::TensorField3DImport()
     , offset_("offset", "Offset", vec3(1.f), vec3(-1000.f), vec3(1000.f), vec3(0.0001f),
```

Comment:
change line 55 to `, inFile_("inFile", "File", "", "tensorfield")`
Then the file dialog will remember the last visited folder for tensor fields

Reply:
Done.